src/OrchardCore.Modules/OrchardCore.Lists/GraphQL/ContainedInputObjectType.cs

```diff
@@ -1,17 +1,19 @@
 using GraphQL.Types;
+using Microsoft.Extensions.Localization;
 using OrchardCore.Apis.GraphQL.Queries;
 using OrchardCore.Lists.Models;
 
 namespace OrchardCore.Lists.GraphQL
 {
     public class ContainedInputObjectType : WhereInputObjectGraphType<ContainedPart>
     {
-        public ContainedInputObjectType()
+        public ContainedInputObjectType(IStringLocalizer<ContainedPart> T)
         {
             Name = "ContainedPartInput";
-            Description = "the list part of the content item";
+            DisplayName = nameof(ContainedPart);
```

Comment:
Remove this for all input types since it doesn't seem to change the default behaviour when not using the field name override.

Reply:
done